torch/nn/utils/rnn.py

```diff
@@ -56,6 +56,7 @@ class PackedSequence(PackedSequence_):
         (i.e., they only pass in tensors conforming to this constraint).
 
     """
+    @ignore
```

Comment:
Why do you need to ignore all of these functions? They were working before?
Also I don't see tests about this?